genomes/models/base_model.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class BaseModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    # created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
```suggestion
```

Reply:
This exists already in the analysis app:
```
class TimeStampedModel(models.Model):
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)

    class Meta:
        abstract = True
```

Reply:
I've removed the redundant genome base model. Resorted to using the Analysis timestamped model